articles/service-fabric/service-fabric-application-and-service-manifests.md

```diff
@@ -94,6 +94,9 @@ For more information on how to configure the SetupEntryPoint, see [Configure the
 
 A Service Fabric Service **Endpoint** is an example of a Service Fabric Resource. A Service Fabric Resource can be declared/changed without changing the compiled code. Access to the Service Fabric Resources that are specified in the service manifest can be controlled through the **SecurityGroup** in the application manifest. When an Endpoint Resource is defined in the service manifest, Service Fabric assigns ports from the reserved application port range when a port isn't specified explicitly. Read more about [specifying or overriding endpoint resources](service-fabric-service-manifest-resources.md).
 
+> [!WARNING]
+> By ServiceFabric design static ports should not conflict with application port range. If you specify a static port, assign it outside of application port range otherwise it can cause port conflicts. With release 6.5CU2 we will issue **Warning** for the same and we may break the application deployment with future releases.
```

Comment:
"..range, otherwise it will result in port conflicts." 

"..issue a Health Warning when we detect such a conflict but let the application deployment continue in sync with the shipped 6.5 behaviour. However, we may prevent application deployment from the next major release.".

Reply:
resolved :)